Update parsing.md (#28736)
DataDog/documentation: content/en/logs/log_configuration/parsing.md

### Matcher and filter

> <div class="alert alert-warning"><strong>Note:</strong> Grok parsing features available at <em>query-time</em> (in <a href="/logs/workspaces/#transformation-cell">Log Workspaces</a> and <a href="/logs/explorer/calculated_fields/">Log Explorer</a>) support a limited subset of matchers (<strong>data</strong>, <strong>integer</strong>, <strong>notSpace</strong>, <strong>number</strong>, and <strong>word</strong>) and filters (<strong>number</strong> and <strong>integer</strong>.)<br><br>

Reviewer: The Note is implied in the warning div

```suggestion
<div class="alert alert-warning">Grok parsing features available at <em>query-time</em> (in <a href="/logs/workspaces/#transformation-cell">Log Workspaces</a> and in the <a href="/logs/explorer/calculated_fields/">Log Explorer</a>) support a limited subset of matchers (<strong>data</strong>, <strong>integer</strong>, <strong>notSpace</strong>, <strong>number</strong>, and <strong>word</strong>) and filters (<strong>number</strong> and <strong>integer</strong>).<br><br>
```

> The full set of matchers and filters listed below is specific to <em>ingest-time</em> <a href="/logs/log_configuration/processors/?tab=ui#grok-parser">Grok Parser</a> functionality.</div>

Reviewer: Changed "below" to "The following..." to remove directional wording.

```suggestion
The following full set of matchers and filters are specific to <em>ingest-time</em> <a href="/logs/log_configuration/processors/?tab=ui#grok-parser">Grok Parser</a> functionality.</div>
```
